packages/scss/src/theming/_commons.scss

```diff
@@ -1,4 +1,8 @@
 $commons: (
+	"animations": (
+		"duration": 300ms,
```

Comment:
it may be relevant to have 2 or 3 animation speed, `fast`, `default/normal`, `slow`